data/src/main/java/com/microsoft/azure/kusto/data/KustoResultSetTable.java

```diff
@@ -288,30 +338,50 @@ public short getShort(String columnName) {
         return (short) get(columnName);
     }
 
+    public short getShortObject(String columnName) {
+        return (Short) get(columnName);
+    }
+
     @Override
     public int getInt(String columnName) {
         return (int) get(columnName);
     }
 
+    public int getInteger(String columnName) {
+        return (Integer) get(columnName);
+    }
+
     @Override
     public long getLong(String columnName) {
         return (long) get(columnName);
     }
 
+    public Long getLongObject(String columnName) {
+        return getLongObject(findColumn(columnName));
```

Comment:
We have no idea what the value type of the value will be, but the getLongObject overload is only intended for an "int".